docs/setup/k8s/running-in-production-k8s.md

```diff
@@ -62,50 +63,65 @@ Below is a quick example of how to setup the a specific stable version of nuclio
-  Stick with this version until you qualify a newer one to work with your system. Since we adhere to backwards compatibility standards between patch versions, and even minor version bumps usually do not break major functionality, the process of qualifying a newer nuclio version should hopefully be short and easy.
-  To version freeze via helm values, set all of the `*.image.repository` and `*.image.tag` configuration keys to the image names and tags which represenr your chosen version's images and are available to your k8s deployment (in case of an [air-gapped installation](#air-gapped-dark-site-operation)).
- 
- 
-## Air gapped (dark site) operation
+<a id="air-gapped-deployment"></a>
+## Air-gapped deployment
 
-We have received various questions about running Nuclio in air gapped environments. Nuclio is fully air-gap compatible, and supports the appropriate configuration to avoid any outside access.
-These guidelines refer to more advanced use cases and they assume the surrounding work (read: devops) can be done by the targeted user.
-We know these things can get a bit tricky. If you want access to a fully-managed, air-gap-friendly, batteries-included, Nuclio deployment, packaged with **Lots** of other goodies - do check out the enterprise grade [Iguazio Data science PaaS](https://www.iguazio.com/platform/)! 
+Nuclio is fully compatible with execution in air-gapped environments ("dark sites"), and supports the appropriate configuration to avoid any outside access.
+The following guidelines refer to more advanced use cases and are based on the assumption that you can handle the related DevOps tasks.
+Note that such implementations can get a bit tricky; to access a fully-managed, air-gap friendly, "batteries-included", Nuclio deployment, which also offers plenty of other tools and features, check out the enterprise-grade [Iguazio Data Science Platform](https://www.iguazio.com/platform/).
+If you select to handle the implementation yourself, follow these guidelines; the referenced configurations are all [Helm values](/hack/k8s/helm/nuclio/values.yaml):
 
-That being said, here are a few guidelines to get you on your way:
+- Set `*.image.repository` and `*.image.tag` to [freeze a qualified version](#version-freezing), and ensure that the configured images are accessible to the Kubernetes deployment.
+- Set `*.image.pullPolicy` to `Never` or to `IfNotPresent` to ensure that Kubernetes doesn't try to fetch the images from the web.
+- Set `offline` to `true` to put Nuclio in "offline" mode.
+- Set `dashboard.baseImagePullPolicy` to `Never`.
+- Set `registry.pushPullUrl` to a registry URL that's reachable from your system.
+- To use a Nuclio templates library (optional), set `dashboard.templatesArchiveAddress` to a location for serving the templates.
```

Comment:
> "location for serving" sounds a bit off
Maybe:
```
To use a Nuclio templates library (optional), you need to package the templates yourself and ensure that they're served locally in a way that's accessible to your system, and set `dashboard.templatesArchiveAddress` to point to where they are served.
```

Reply:
I changed it to the following (I took my cue also from the value name) — see commit dfd8747:
> - To use a Nuclio templates library (optional), package the templates into an archive that's served locally and accessible to your system, and set `dashboard.templatesArchiveAddress` to the archive address

Reply:
That is a bit inaccurate as "archive" doesn't have an address. it is served on some server that serves it on some address (we're talking http serving here)